en.rss

```diff
@@ -8,14 +8,37 @@
 		<copyright>Alexander Matheisen</copyright>
 		<pubDate>Sun, 28 Apr 2013 14:11:28 +0200</pubDate>
 		<atom:link href="https://blog.openrailwaymap.org/en.rss" rel="self" type="application/rss+xml"/>
-		<lastBuildDate>Wed, 28 Feb 2018 23:40:00 +0100</lastBuildDate>
+		<lastBuildDate>Sun, 15 Oct 2023 21:40:00 +0200</lastBuildDate>
 		<sy:updatePeriod>daily</sy:updatePeriod>
 		<sy:updateFrequency>1</sy:updateFrequency>
 		<image>
 			<url>https://www.openrailwaymap.org/img/openrailwaymap-310.png</url>
 			<title>OpenRailwayMap</title>
 			<link>https://blog.openrailwaymap.org/</link>
 		</image>
+		<item>
+			<title>Improved search function and new API</title>
+			<description>OpenRailwayMap has got an improved search function based on a new API.</description>
+			<link>https://blog.openrailwaymap.org?lang=en#34</link>
+			<author>Michael Reichert</author>
+			<dc:creator>Michael Reichert</dc:creator>
+			<guid>https://blog.openrailwaymap.org?lang=en#34</guid>
+			<pubDate>Sun, 15 Oct 2023 21:40:00 +0200</pubDate>
+			<content:encoded>
+				<![CDATA[
+					<div class="uk-grid">
+						<div class="uk-width-medium-1-2">
+							<p>The search function of OpenRailwayMap uses from now on a new implementation.</p>
+							<p>When searching for stations, previously only the <code>name=*</code> tag of OpenStreetMap was taken into account. From now on, all variants of the <code>name=* tag</code>, i.e. shortened, official and foreign-language names are used.</p>
+							<p>In the past, the string entered in the search field had to be equal with the beginning of the name of the object. Instead, the new search uses a full-text index. Skipping the prefix of a station name (suburban stations without the name of the city, e.g. Spandau instead of Berlin-Spandau) will not exclude the station from appearing in the search results.</p>
```

Comment:
"had to be equal with" -> "had to match"

"will not exclude" -> "will not prevent"